src/components/tabs/Tabs.module.css

```diff
@@ -0,0 +1,22 @@
+.container {
+	padding-block: 1rem;
+}
+
+.tablist {
+	display: flex;
+}
+
+.tab {
+	border-radius: unset;
+	background-color: unset;
+	padding: 0.5rem 1rem;
+	border-bottom: 3px solid var(--theme-bg-accent);
```

Comment:
The other tabs are using `--theme-divider` for the color and 4px as its size, I think it would be better if we used the same values for all the Tabs. Also, maybe we should expand the divider line to fill the whole screen as the Installation Tab does?

I'd also add `cursor: pointer` just so it's clear we can click the tab

```suggestion
	border-bottom: 4px solid var(--theme-divider);
	cursor: pointer;
```

Reply:
Sure, we could extend the divider! Agreed on the suggestion too 👍